### PR TITLE
Implement about page

### PR DESCRIPTION
**Pull request by anjali-dhanuka**
Hey @tapasweni-pathak 
The link to "About" page has been removed  referring to the issue https://github.com/systers/vms/issues/403.
Please review.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ coveralls==0.5
 Django==1.7.4
 docopt==0.6.2
 flake8==3.4.1
-psycopg2==2.5.4
+psycopg2==2.7.3
```
Comment (tapasweni-pathak) [inline requirements.txt]: did you test w/ fresh setup after this?

**Comment by anjali-dhanuka** [comment]
Hey @tapasweni-pathak I have squashed all commits and restored secret key as you stated. Please review.

File: requirements.txt (hunk 1 above)
Comment (anjali-dhanuka) [reply requirements.txt]: Yes, it states that django secret key is to be used in place of secret key.

**Comment by anjali-dhanuka** [comment]
@tapasweni-pathak it uses latest psql i.e version 10.1 which has compatibility issues with psycopg2.5.3

**Pull request merged**